object-store/src/main/java/org/hypertrace/config/objectstore/DeletedContextualConfigObjectImpl.java

```diff
@@ -7,13 +7,13 @@
 import org.hypertrace.config.service.v1.ContextSpecificConfig;
 
 @lombok.Value
-class DeletedContextualConfigObjectImpl<T> implements DeletedContextualConfigObject<T> {
+public class DeletedContextualConfigObjectImpl<T> implements DeletedContextualConfigObject<T> {
   String context;
   Optional<T> deletedData;
   Instant creationTimestamp;
   Instant lastUpdatedTimestamp;
 
-  DeletedContextualConfigObjectImpl(
+  public DeletedContextualConfigObjectImpl(
```

Comment:
Had to make this public as having to build the object explicitly in some scenarios where delete object method has been overridden.

Reply:
Not following - shouldn't be public. If there's cases where we're trying to override this (like in mocks?) they can implement (or mock) the interface.

Reply:
Got it. Updated now.